ocaml-lsp-server.opam.template

```diff
@@ -1,3 +1,16 @@
+depends: [
+  "csexp" {>= "1.2.3"}
+  "dune-build-info"
+  "dune" {>= "2.5.0"}
+  "menhir"
+  "ocamlfind" {>= "1.5.2"}
+  "ocamlformat" {with-test}
+  "ppx_yojson_conv_lib"
+  "result" {>= "1.5"}
+  "stdlib-shims"
+  "yojson"
+  ("ocaml" {>= "4.06" & < "4.12"} | ("ocaml" {< "4.08"} & "ocamlfind-secondary"))
```

Comment:
If you don't want to move everything to a template, we can install the `ocamlfind-secondary` only in the Actions runner.

Reply:
This is just affected by the setup-ocaml spec, so I decided to handle it only in the Actions runner at the moment.